### PR TITLE
Allow to delete own motions

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
Can be tested, just the validationerror has to be changed.

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
@normanjaeckel Please review later. I have to merge it today. Thanks.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel see #3518 .

File: openslides/motions/views.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def check_view_permissions(self):
         """
         if self.action in ('list', 'retrieve'):
             result = self.get_access_permissions().check_permissions(self.request.user)
-        elif self.action in ('metadata', 'partial_update', 'update'):
+        elif self.action in ('metadata', 'partial_update', 'update', 'destroy'):
```
Comment (normanjaeckel) [inline openslides/motions/views.py]: Please extend comment in line below.

File: openslides/motions/views.py [hunk 2]
```diff
@@ -85,6 +85,19 @@ def check_view_permissions(self):
...
         return result
 
+    def destroy(self, request, *args, **kwargs):
+        """
+        Destroy is allowed if the user has manage permissions, or he is the submitter and
+        the current state allows to edit the motion.
+        """
+        motion = self.get_object()
+
+        if (has_perm(request.user, 'motions.can_manage') or
```
Comment (normanjaeckel) [inline openslides/motions/views.py]: Please switch if condition:
```
if <not allowed>:
    raise ...
return super()....
```

**Pull request merged**